tests/integration/test_post_endpoint.py

```diff
@@ -53,7 +53,7 @@ async def test_post_secrets(
     response = client.post(url="/secrets", json=request_body)
     assert response.status_code == 200
     body = response.json()
-    secret = base64.b64decode(body["secret"])
+    submitter_secret = base64.b64decode(body["submitter_secret"])
```

Comment:
Maybe add a check for the presence of the new key?

Reply:
Good point.